src/test/java/uk/gov/hmcts/reform/sandl/snlrules/utils/DateTimeUtilsTests.java

```diff
@@ -108,4 +108,54 @@ public void should_not_be_between_when_4_and_7() {
             4, 7
         ));
     }
+
+    @Test
+    public void should_be_older_than_10_days() {
+        OffsetDateTime dateToBeChecked = OffsetDateTime.of(2018, 05, 10, 9, 0, 0, 0, ZoneOffset.UTC);
+        Assert.assertTrue(DateTimeUtils.olderThan(
+            dateToBeChecked, 2018, 05, 21, 10));
+    }
+
+    @Test
+    public void should_not_be_older_than_10_days() {
+        OffsetDateTime dateToBeChecked = OffsetDateTime.of(2018, 05, 10, 9, 0, 0, 0, ZoneOffset.UTC);
+        Assert.assertFalse(DateTimeUtils.olderThan(
+            dateToBeChecked, 2018, 05, 20, 10));
+    }
+
+    @Test
+    public void should_be_between_two_dates() {
+        OffsetDateTime dateToBeChecked = OffsetDateTime.of(2018, 05, 16, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime after = OffsetDateTime.of(2018, 05, 10, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime before = OffsetDateTime.of(2018, 05, 21, 9, 0, 0, 0, ZoneOffset.UTC);
+
+        Assert.assertTrue(DateTimeUtils.between(dateToBeChecked, after, before));
+    }
+
+    @Test
+    public void should_not_be_between_two_dates() {
+        OffsetDateTime dateToBeChecked = OffsetDateTime.of(2018, 05, 30, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime after = OffsetDateTime.of(2018, 05, 10, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime before = OffsetDateTime.of(2018, 05, 21, 9, 0, 0, 0, ZoneOffset.UTC);
+
+        Assert.assertFalse(DateTimeUtils.between(dateToBeChecked, after, before));
+    }
+
+    @Test
+    public void should_be_between_two_dates_when_on_date() {
+        OffsetDateTime dateToBeChecked = OffsetDateTime.of(2018, 05, 21, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime after = OffsetDateTime.of(2018, 05, 10, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime before = OffsetDateTime.of(2018, 05, 21, 9, 0, 0, 0, ZoneOffset.UTC);
+
+        Assert.assertTrue(DateTimeUtils.between(dateToBeChecked, after, before));
+    }
+
+    @Test
+    public void should_be_between_two_dates_when_on_date_but_different_hours() {
+        OffsetDateTime dateToBeChecked = OffsetDateTime.of(2018, 05, 21, 12, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime after = OffsetDateTime.of(2018, 05, 10, 9, 0, 0, 0, ZoneOffset.UTC);
+        OffsetDateTime before = OffsetDateTime.of(2018, 05, 21, 9, 0, 0, 0, ZoneOffset.UTC);
+
+        Assert.assertTrue(DateTimeUtils.between(dateToBeChecked, after, before));
+    }
```

Comment:
a test that covers fact that before > after - what would it do?

Reply:
done, exception was added